kahuna/public/stylesheets/main.css

```diff
@@ -518,6 +518,15 @@ input.search-query__input {
     display: none;
 }
 
+.file-uploader__select-files {
+    padding: 5px 10px;
+}
+
+@media screen and (max-width: 1200px) {
```

Comment:
Perhaps simply organising the contents of the main.css file in the 1st way you describe would suffice?